community.html

```diff
@@ -37,86 +39,89 @@ <h2 property="schema:name">Where to collaborate</h2>
                 <li>
                   <p><strong>Solid Project Forum</strong></p>
                   <p>Create and respond to comments, questions, and discussions happening in the community.</p>
-                  <a href="" rel="rdfs:seeAlso">Go to Solid Forum</a>
+                  <a href="https://forum.solidproject.org" rel="rdfs:seeAlso">Go to Solid Forum</a>
                 </li>
                 <li>
                   <p><strong>Solid Matrix Chat</strong></p>
                   <p>The Solid Matrix Chat is a live chat where community members can connect.</p>
-                  <a href="" rel="rdfs:seeAlso">Go to Matrix</a>
+                  <a href="https://matrix.to/#/#solid:converser.eu" rel="rdfs:seeAlso">Go to Matrix</a>
                 </li>
                 <li>
                   <p><strong>Solid Community Group</strong></p>
                   <p>Learn and work on Solid specifications.</p>
-                  <a href="" rel="rdfs:seeAlso">Go to Community Group</a>
+                  <a href="https://www.w3.org/groups/cg/solid/" rel="rdfs:seeAlso">Go to Community Group</a>
                 </li>
               </ul>
             </div>
           </section>
 
           <section id="events" inlist="" rel="schema:hasPart" resource="#events">
-            <h2 property="schema:name">Events</h2>
             <div datatype="rdf:HTML" property="schema:description">
-              <p>Discover Solid events taking place virtually and/or in person around the world.</p>
+              <header>
+                <h2 property="schema:name">Events</h2>
+                <p>Discover Solid events taking place virtually and/or in person around the world.</p>
+              </header>
+              <article>
+                <ul id="events-list">
+                  <li id="events-2023" rel="schema:hasPart" resource="#events-2023">
+                    <details open="">
+                      <summary property="schema:name">2023 Events</summary>
+                      <ul rel="schema:hasPart">
+                        <li>
+                          <dl about="#mydata-2023-conference" id="mydata-2023-conference" rel="schema:hasPart" typeof="schema:Event">
+                            <dt>Title</dt>
+                            <dd property="schema:name"><a href="https://2023.mydata.org/" rel="schema:url">MyData 2023 Conference</a></dd>
+                            <dt>Organization</dt>
+                            <dd><a href="https://mydata.org/" rel="schema:organizer">MyData</a></dd>
+                            <dt>Date</dt>
+                            <dd><time property="schema:startDate" datatype="xsd:date" content="2023-05-31" datetime="2023-05-31">2023-05-31</time>–<time property="schema:endDate" datatype="xsd:date" content="2023-06-02" datetime="2023-06-02">2023-06-02</time></dd>
```

Comment:
I know I included `content` in the original but it is not necessary if the date is always going to be yyyy-mm-dd without time. The precision is the same. Value from the text content will be used. If the precision for the machine-readable is desired, and the human gets less precise in text content, then `content` can be handy.

```suggestion
                            <dd><time property="schema:startDate" datatype="xsd:date" datetime="2023-05-31">2023-05-31</time>–<time property="schema:endDate" datatype="xsd:date" datetime="2023-06-02">2023-06-02</time></dd>
```

Not sure if you want to follow this pattern for the rest.

Reply:
I have no strong opinions on this, but in general less is better. Do you want me to remove all uses of `content="[date]"` in the HTML?